runtime/compiler/ilgen/Walker.cpp

```diff
@@ -5323,6 +5323,27 @@ break
          }
          _intrinsicErrorHandling = false;
       }
+
+   // The call may be transformed into a non-OSR point. Check if bookkeeping is needed
+   // before the transformation.
+   bool needOSRBookkeeping = false;
+   int32_t osrInductionOffset;
+
+   // callNodeTreeTop may be null if this call should not be placed in the trees
+   if (callNodeTreeTop
+       && comp()->getOption(TR_EnableOSR)
+       && !comp()->isPeekingMethod()
+       && comp()->isOSRTransitionTarget(TR::postExecutionOSR)
```

Comment:
what about if the mode is preExecutionOSR do we need another helper marker for that?